main.js

```diff
@@ -31,3 +32,299 @@ mobileIcons.addEventListener('click', toggleMenuOptions);
+      featuredImage: './images/mobile-featured-image.svg',
+      technologies: ['Ruby on rails', 'css', 'JavaScript'],
+      liveVersionLink: 'https://dd-obua.github.io/Dan-portfolio/',
+      sourceLink: 'https://github.com/dd-obua/Dan-portfolio',
+    },
+  },
+];
+
+const modalContainer = document.getElementById('modal-desktop');
+const mobileModalContainer = document.getElementById('modal-mobile');
+const closePopup = () => {
+  const closeModalIcon = document.querySelector('.close-modal-icon');
+  closeModalIcon.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (!modalContainer.classList.contains('hide-modal')) {
+      modalContainer.classList.add('hide-modal');
+    }
+  });
+};
+
+const closeMobilePopup = () => {
+  const closeMobileModalIcon = document.querySelector('.close-mobile-modal-icon');
+  closeMobileModalIcon.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (!mobileModalContainer.classList.contains('hide-modal')) {
+      mobileModalContainer.classList.add('hide-modal');
+    }
+  });
+};
+
+const multiStories = document.querySelector('.multi-stories');
+stories.forEach((stry, index) => {
+  let storyItems = '';
+  stry.languages.map((storyItem) => {
+    storyItems += ` <li>${storyItem}</li>`;
+    return null;
+  });
+
+  const story = `
+<div class="story">
+<div class="empty"></div>
+          <div class="content">
+            <h2> ${stry.storyTitle} </h2>
+            <ul> ${storyItems} </ul>
+            <button class="btn" id="${index}" type="button" name="story-button" aria-label="See project button" >See Project</button>
+          </div>
+        </div>
+`;
+
+  multiStories.innerHTML += story;
+});
+
+function displayModal() {
+  const storyBtns = document.querySelectorAll('.btn');
+  storyBtns.forEach((storyBtn) => {
+    storyBtn.addEventListener('click', () => {
+      const btnId = storyBtn.getAttribute('id');
+      const storyInfo = stories[btnId];
+      if (window.innerWidth >= 768) {
+        let technologies = '';
+        storyInfo.storyDetails.technologies.map((technology) => {
+          technologies += `<li>${technology}</li>`;
+          return null;
+        });
+        const modalTemplate = `
+          <div class="modal">
+            <div class="desktop-modal-content">
+              <i class="close-modal-icon fa-solid fa-xmark fa-xl"></i>
+              <img src="${storyInfo.storyDetails.featuredImage}" class="img" alt="" />
+              <div class="below">
+                <h3> ${storyInfo.storyDetails.name} </h3>
+                  <a href="${storyInfo.storyDetails.liveVersionLink}">See Live <img src="./images/live.svg" alt="" /></a>
+                  <a href="${storyInfo.storyDetails.sourceLink}">See Source <img src="./images/source.svg" alt="" /></a>
+              </div>
+              <ul> ${technologies} </ul>
+              <p> ${storyInfo.storyDetails.description} </p>
+            </div>
+          </div>
+        `;
```

Comment:
- [ ] [Required] As the previous reviewer suggested, I also kindly suggest that you make the page background blur when a modal is displayed. 

### EXAMPLE
.body {
  backdrop-filter: blur(6px);
}

- [ ] [OPTIONAL] Add an overflow: hidden style to the page when a modal is on display so that the user does not accidentally scroll away from the current project.